apps/api/src/config/env-validator.ts

```diff
@@ -44,6 +54,8 @@ const validators: { [K in keyof any]: ValidatorSpec<any[K]> } = {
   NEW_RELIC_LICENSE_KEY: str({
     default: '',
   }),
+  REDIS_CACHE_HOST: recommended({ default: '' }),
+  REDIS_CACHE_PORT: recommended({ default: '' }),
```

Comment:
Shouldn't be mandatory? For example if not set tests would fail in any environment.

Reply:
@p-fernandez So this one is an open question for me because I am not sure if it should be mandatory for our self-hosted users. LMKWYT

Reply:
Well if the environment variables are different. and the user provided only REDIS_HOST, REDIS_PORT without the  REDIS_CACHE_HOST  REDIS_CACHE_PORT,  the cache service won't initialize and the interceptor will skip the caching logic. (but you are right regarding the test we need to add a flag there as well 😀)

Reply:
Ah good that will skip the caching logic. Nothing to say then. 🙂